### PR TITLE
Reformat some code for readability and prettying

### PR DESCRIPTION
**Pull request by madeddy**
- reordering of some argparse args
- add py-version guard
- max-line-lenght 100
- move sharelock alongside Lock()

Argparse reordering logic is up -> down from often used/important -> other.

I've foregone the separating into own func as you where not really comfortable with it and its not a must have for me. I do this mostly if argparse gets as big as here and i want main() often small and more onto topic.

I moved also sharelock to his pal as it looked so alone down there. 😂

### CODE REVIEW

File: unrpyc.py [hunk 1]
```diff
@@ -187,64 +193,116 @@ def worker(t):
...
+        "disabled. The generated code is exactly equivalent, only slightly more cluttered.")
+
+    ap.add_argument(
+        '--register-sl-displayable',
+        dest="sl_custom_names",
+        type=str,
+        nargs='+',
+        help="Accepts mapping separated by '=', "
+        "where the first argument is the name of the user-defined displayable object, "
+        "and the second argument is a string containing the name of the displayable,"
```
Comment (CensoredUsername) [inline unrpyc.py:301]: This is missing a space at the end. might've been that way originally, but mind fixing it as well?

File: unrpyc.py [hunk 2]
```diff
@@ -187,64 +193,116 @@ def worker(t):
...
+
+    ap.add_argument(
+        '-p',
+        '--processes',
+        dest='processes',
+        action='store',
+        type=int,
+        choices=list(range(1, cc_num)),
+        default=cc_num - 1 if cc_num > 2 else 1,
+        help="Use the specified number or processes to decompile."
```
Comment (CensoredUsername) [inline unrpyc.py:241]: Also missing a space at the end

Comment (madeddy) [reply unrpyc.py:241]: k

File: unrpyc.py (hunk 1 above)
Comment (madeddy) [reply unrpyc.py:301]: I let the one in L302 as i thought you want propably the parens direct on the word.

Comment (CensoredUsername) [reply unrpyc.py]: Ah, I just missed that one 😅

**Comment by CensoredUsername** [comment]
Can't complain about formatting fixes, merged!

**Pull request merged**